keyboards/ai03/jp60/readme.md

```diff
@@ -0,0 +1,27 @@
+# JP60
+
+![JP60](https://github.com/ai03-2725/JP60/raw/main/Render/Front.png)
+
+GH60-compatible JIS-like hotswap keyboard PCB
+
+* Keyboard Maintainer: [ai03](https://github.com/ai03-2725)
+* Hardware Supported: [JP60 PCB](https://github.com/ai03-2725/JP60)
+* Hardware Availability: To be made available in JP market soon; will be listed in the repo above
+
+Make example for this keyboard (after setting up your build environment):
+
+    make jp60:default
```

Comment:
```suggestion
    make ai03/jp60:default
```

Reply:
Whoops. No clue how I missed this.
Fixed with [e558964](https://github.com/qmk/qmk_firmware/pull/14219/commits/e558964fd92bbf655aabe4414929ed96d569a4bb).